near-sdk-macros/src/lib.rs

```diff
@@ -316,3 +359,93 @@ pub fn function_error(item: TokenStream) -> TokenStream {
         }
     })
 }
+
+/// Adding the `events` argument for the `#[near_bindgen]` macro injects this derive macro and should be the used instead of this
```

Comment:
It was hard to understand what you mean here. I hope I managed to unpack it correctly:

```suggestion
/// NOTE: This is an internal implementation for `#[near_bindgen(events(standard = ...))]` attribute.
```